src/input_xml.F90

```diff
@@ -2431,8 +2434,10 @@ subroutine read_tallies_xml()
 
           ! Determine number of bins
           if (check_for_node(node_filt, "bins")) then
-            if (trim(temp_str) == 'energy' .or. &
-                 trim(temp_str) == 'energyout') then
+            if ((temp_str == 'energy' .or. &
+                 temp_str == 'energyout') .or. &
+                 (temp_str == 'mu' .or. temp_str == 'polar') .or. &
+                 (temp_str == 'azimuthal')) then
```

Comment:
three pairs of extra parentheses in this expression. Sorry, should've pointed that out the first time around.